### ISSUE TITLE
Provide firmware HEX image

### ISSUE THREAD

**Issue by jpmeijers** [SeongJiIoT/LSM100A_SDK#2]
Could you please provide a BIN or HEX of the final firmware to program into the module?

**Comment by G-MatrixSystems** [comment]
Yes, Please supply final firmware hex file as a release.

**Comment by jpmeijers** [comment]
Yes definitely, @SeongJiIoT please provide the final firmware HEX file under the GitHub Releases section:
https://github.com/SeongJiIoT/LSM100A_SDK/releases

It needs to be the final one I can program using STM32CubeProgrammer.

**Comment by SeongJiIoT** [comment]
We do not provide firmware other than IAP (Bootloader) FW on Github according to our policy.

Refer to chapter 1 of the FW download guide document for the firmware (hex file) created by building the LSM100A SDK with the STCubeIDE tool to create the final firmware.

The generated firmware name must be modified in the batch file and merged.

After downloading the FW with IAP (Bootloader), you can download the bin file as Serial (Y-modem).

**Comment by jpmeijers** [comment]
That is impossible to do. Please see my other issue: https://github.com/SeongJiIoT/LSM100A_SDK/issues/3

**Comment by jpmeijers** [comment]
> We do not provide firmware other than IAP (Bootloader) FW on Github according to our policy.

Please provide it somewhere else then.

**Comment by SeongJiIoT** [comment]
Without merging, would you like to download the LSM_IAP_V100_220516.hex IAP (Bootloader) file and the Hex file created by building, respectively, to STCubeProgrammer?
However, you should never delete the entire memory.

**Comment by jpmeijers** [comment]
> Without merging, would you like to download the LSM_IAP_V100_220516.hex IAP (Bootloader) file and the Hex file created by building, respectively, to STCubeProgrammer? However, you should never delete the entire memory.

I'm confused by your question. I just want to update the firmware of my EVB-LSM without having to install and learn to use the STM toolchain. Provide us with the final firmware hex image please.

**Comment by SeongJiIoT** [comment]
If you contact us through our sales team, we will send you a link to download the firmware by e-mail.

**Comment by jpmeijers** [comment]
> If you contact us through our sales team, we will send you a link to download the firmware by e-mail.

I did an hour ago. Still waiting for a response.

**Comment by jpmeijers** [comment]
Ahh excellent! Thanks a lot @NemesisXB.

I still have not heard anything back from my query via the sales channels @SeongJiIoT. If this is the kind of support I should expect I think I should rather look further for a different vendor.

**Comment by jpmeijers** [comment]
@NemesisXB thanks, your firmware image is loaded and running.

Currently I am talking to [Mipot](https://mip.mipot.com/). Also has local distributors, answers my support queries immediately, and doesn't hide the fact that all the features aren't ready yet. They're slightly more expensive, but I'm willing to pay for that support and reliability. I really can't deal with things like 
> We do not \<want to help you\> according to our policy.

:roll_eyes:

**Comment by SeongJiIoT** [comment]
The firmware request was delivered through the sales team.
We will send you the firmware link by e-mail.

**Comment by jpmeijers** [comment]
> The firmware request was delivered through the sales team. We will send you the firmware link by e-mail.

https://drive.google.com/file/d/1_d21Wp7V_2SuZFKTE2mid31bN9PyN9Y8/view?usp=sharing

@NemesisXB let's do a binary diff and see if they are the same.

**Comment by NemesisXB** [comment]
The binary files do not match, but they might have used a different version compiler than what I used.
